modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets_{context}"]
+= Creating a secret using the web console
+
+You can create secrets using the web console. 
+
+.Procedure
+
+. Navigate to *Workloads* -> *Secrets*.
+. Click *Create* -> *From YAML*.
+.. Edit the YAML manually to your specifications, or drag and drop a file into the YAML editor. 
+For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+    name: example
+    namespace: <namespace>
+type: Opaque <1>
+data:
+    username: <username>
+    password: <password>
+stringData: <2>
+    hostname: my app.mydomain.com 
```

Comment:
Some nits, otherwise LGTM:
- indent with two spaces for each newly nested filed. Four spaces work but not conventional.
- Under `data`, the fields' values are base64 encoded instead of raw values.
- `my app` has a typo of an unneeded space

```suggestion
metadata:
  name: example
  namespace: <namespace>
type: Opaque <1>
data:
  username: <base64 encoded username>
  password: <base64 encoded password>
stringData: <2>
  hostname: myapp.mydomain.com 
```